fastlane/Fastfile

```diff
@@ -16,6 +16,10 @@ APPS = {
     share_to_provisioning_profile: "2bbfa6d3-0e13-4a69-ab75-07fb76dc8a49",
     share_to_provisioning_profile_name: "match AppStore com.cliqz.ShareTo",
 
+    open_in_identifier: "com.cliqz.ShareTo",
```

Comment:
open_in_identifire shouldn't be "com.cliqz.OpenIn"?

Reply:
fixed